workspaces/graphiql/plugins/graphiql/README.md

```diff
@@ -64,6 +65,8 @@ If all you need is a static list of endpoints, the plugin exports a `GraphQLEndp
 +               body: JSON.stringify(params),
 +             }).then(res => res.json());
 +           },
++           // Optional list of plugins
++           plugins: [explorerPlugin()]
```

Comment:
perhaps this would be enough? Otherwise it looks like installing the package is required

```suggestion
+           plugins: []
```

Reply:
Sure! Added the Explorer plugin just for the sake of an example.